To Do App/script.js

```diff
@@ -0,0 +1,37 @@
+const form = document.querySelector("form");
+const input = document.querySelector("#txtTaskName");
+const btnDeleteAll = document.querySelector("#btnDeleteAll");
+const taskList = document.querySelector("#task-list");
+//Call Event Listener
+eventLiteners();
+function eventLiteners() {
+    // submit event
+    form.addEventListener("submit", addNewItem);
+};
+
+//Add new item
+function addNewItem(e) {
+    if (input.value === "") {
```

Comment:
short if diye bir sey var. Daha kolay yazmasi ve okunakli. Arastirabilirsin. 
```suggestion
     (input.value === "")  ? alert("add new item") : "";
```

Reply:
short if buna bakılacak.